cql3/constants.hh

```diff
@@ -73,7 +73,7 @@ public:
         value(cql3::raw_value bytes_) : _bytes(std::move(bytes_)) {}
         virtual cql3::raw_value get(const query_options& options) override { return _bytes; }
         virtual cql3::raw_value_view bind_and_get(const query_options& options) override { return _bytes.to_view(); }
-        virtual sstring to_string() const override { return to_hex(*_bytes); }
+        virtual sstring to_string() const override { return _bytes.to_view().with_linearized([] (bytes_view bv) { return to_hex(bv); }); }
```

Comment:
Can't we use `with_value` here to avoid a new call to `with_linearized?

Reply:
We would need a generalized `to_hex`. Okay, I will do that.

Reply:
Done.